src/sdk/namespace_fs.js

```diff
@@ -682,7 +682,49 @@ class NamespaceFS {
         const fs_account_config = this.set_cur_fs_account_config(object_sdk);
         await this._load_bucket(params, fs_account_config);
 
+        if (typeof nb_native().fs.temp_file === 'function') {
```

Comment:
Based on the [this suggestion](https://github.com/noobaa/noobaa-core/pull/6947/files#r888402205), change this to:
```suggestion
        if (nb_native().fs.gpfs) {
```

Reply:
oh I see that you wanted this to represent a generic capability to use temp_file, instead of checking for backend type, but I prefer to keep most logic in the js world, and just provide a straightforward api to the native functions. if we need to support more backends with temp file we can fix it up here to allow more backends to use the same flow.